### PR TITLE
refresh the docs

### PR DESCRIPTION
**Pull request by kilbergr**
Upon doing further work on PagerDuty, I realized the docs could once again be clarified. Ever closer to a high-fidelity representation.

### CODE REVIEW

File: infrasec/aws/sns-guardduty-alert-integrations.md [hunk 1]
```diff
@@ -229,7 +250,8 @@ resource "pagerduty_service_integration" "guardduty" {
 ### Subscription
 
 ```hcl
-resource "aws_sns_topic_subscription" "team_notification_region" {
+resource "aws_sns_topic_subscription"
+"guardduty_team_notification_region" {
```
Comment (brainsik) [inline infrasec/aws/sns-guardduty-alert-integrations.md:254]: Is this intentionally on it's own line instead of with the `resource` line above?

**Review by brainsik — APPROVED** [review]
Approved with a question

File: infrasec/aws/sns-guardduty-alert-integrations.md (hunk 1 above)
Comment (kilbergr) [reply infrasec/aws/sns-guardduty-alert-integrations.md:254]: :) No

Comment (kilbergr) [inline infrasec/aws/sns-guardduty-alert-integrations.md:254]:
```suggestion
resource "aws_sns_topic_subscription" "guardduty_team_notification_region" {
```

**Pull request merged**